src/Compilers/Core/Portable/InternalUtilities/Debug.cs

```diff
@@ -18,7 +18,13 @@ internal static class RoslynDebug
 
         /// <inheritdoc cref="Debug.Assert(bool, string)"/>
         [Conditional("DEBUG")]
-        public static void Assert([DoesNotReturnIf(false)] bool condition, string message)
+        public static void Assert([DoesNotReturnIf(false)] bool condition,
```

Comment:
>Assert [](http://example.com/codeflow?start=27&length=6)

Do we really need to maintain a private version of Assert? I was under impression that standard ```Debug.Assert``` has been appropriately decorated.

Reply:
Agreed with this. I thought the main reason we still kept `RoslynDebug.Assert` around was that we haven't removed all of the callsites.